syne_tune/remote/remote_launcher.py

```diff
@@ -213,8 +213,8 @@ def launch_tuning_job_on_sagemaker(self, wait: bool):
             instance_type=self.instance_type,
             instance_count=1,
             role=self.role,
-            py_version="py3",
-            framework_version='1.6',
+            py_version="py38",
+            framework_version='1.10.0',
             image_uri=self.syne_tune_image_uri(),
             hyperparameters=hyperparameters,
             checkpoint_s3_uri=checkpoint_s3_root,
```

Comment:
Yes it does, I forgot about this one as well but I can think I can fix it as well without much change.

Regarding refactoring the RL so that it can a sequence of experiments, I would prefer to continue using the two script approach as it would be likely hard to maintain a RL that can either run one experiment or many.